spec/traits.dd

```diff
@@ -1713,6 +1714,39 @@ $(H2 $(GNAME toType))
 
         $(RATIONALE Provides the inverse operation of the $(DDSUBLINK property, mangleof, `.mangleof`) property.)
 
+$(H2 $(GNAME initSymbol))
+
+        $(P Takes a single argument, which must evaluate to a `class` or `struct` type.
+            Returns a `const(void)[]` that points to either the initializer symbol of the supplied
+            type or `null` if it is $(RELATIVE_LINK2 isZeroInit, zero-initialized). The referenced symbol
```

Comment:
`__traits(isZeroInit, <class>)` is always true (=> class *ref*). All classes have init symbols (vptr is definitely not null).

Reply:
Right. Removed the link